sensor/kubernetes/localscanner/cert_refresher.go

```diff
@@ -37,6 +38,8 @@ type serviceCertificatesRepo interface {
 	getServiceCertificates(ctx context.Context) (*storage.TypedServiceCertificateSet, error)
 	// ensureServiceCertificates persists the certificates on permanent storage.
 	ensureServiceCertificates(ctx context.Context, certificates *storage.TypedServiceCertificateSet) error
+	// ensureServiceCertificates returns map of supported service type certificates
+	getKnownServiceTypes() map[storage.ServiceType]serviceCertSecretSpec
```

Comment:
Given the name of this method I would have expected the method to really only return service types in some data structure (set, frozen set, slice, etc.), but instead you are returning much more. I find this a bit confusing and maybe unnecessary?